tests/cupy_tests/padding_tests/test_pad.py

```diff
@@ -10,7 +10,8 @@
 
 @testing.parameterize(
     *testing.product({
-        'array': [numpy.arange(6).reshape([2, 3])],
```

Comment:
I think you mixed two PRs?😅

Reply:
right. I was going to enable `mode='median'` in cupy.pad and accidentally checked that in here.

Reply:
😁